normalize_test.go

```diff
@@ -0,0 +1,33 @@
+package errors
+
+import (
+	"regexp"
+	"testing"
+)
+
+func errorMatches(t *testing.T, err error, re string) {
+	if err == nil && re != "" {
+		t.Errorf("nil error doesn't match %s", re)
+		return
+	}
+	match, reErr := regexp.MatchString(re, err.Error())
```

Comment:
why use a regexp when the test only checks for literal strings 🤔.

Reply:
At line 32, regexp was used for check whether the error string was exactly the pattern.

```
errorMatches(t, notWrapped, `^\[Internal:Test\]everything is alright$`)
```

Reply:
@YuJuncen  isn't line 29 also checking for exact match 🤔

Reply:
> MatchString reports whether the string s contains any match of the regular expression pattern. More complicated queries need to use Compile and the full Regexp interface.

@kennytm It matches any substring.

Reply:
Anyway, this might be useful in the future... (I hope...)